src/ResourceManager/Resources/Commands.Resources/Models.ResourceGroups/ResourceClient.cs

```diff
@@ -313,16 +313,28 @@ public static List<string> ParseDetailErrorMessage(string statusMessage)
         {
             DeploymentExtended deployment;
 
+            // Poll deployment state and deployment operations.
```

Comment:
@bganapa the changes in this file are already included in [this PR](https://github.com/Azure/azure-powershell/pull/4105). Should we close the other PR and include the changes here?